tasks/main.yml

```diff
@@ -51,6 +52,7 @@
   apt:
     name: dokku
     install_recommends: yes
+    update_cache: yes
```

Comment:
[apt_repository module](https://docs.ansible.com/ansible/latest/modules/apt_repository_module.html) has `update_cache`, and the default value is yes.
So I do not want to `update_cache: yes` is here.

Reply:
I'm using [apt module](https://docs.ansible.com/ansible/latest/modules/apt_module.html) (not the same as [apt_repository module](https://docs.ansible.com/ansible/latest/modules/apt_repository_module.html)) who have `update_cache` default value to no.

Reply:
My intentional is updating cache when adding repo by `apt_repository` module above, and each apt module does not update cache.
If I use `update_cache`, I want to use `cache_valid_time` too.